### ISSUE TITLE
Git Stage Changes keybinding does not work in insiders

### ISSUE THREAD

**Issue by albireox** [Microsoft/vscode#50821]
Issue Type: <b>Bug</b>

1. Install VSCode Insiders. Launch without extensions.
2. Assign a shortcut to `Git Stage Changes`
3. Try to stage a change using the shortcut.
4. This seems to work in the stable VSCode.

VS Code version: Code - Insiders 1.24.0-insider (43d168182c8df35a99c7f5ee0ef227de34f75fbd, 2018-05-30T07:06:30.237Z)
OS version: Darwin x64 17.5.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-5557U CPU @ 3.10GHz (4 x 3100)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>vpx_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|16.00GB (1.18GB free)|
|Process Argv|/Applications/Visual Studio Code - Insiders.app/Contents/MacOS/Electron|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (20)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-simple-icons|Lau|1.8.3
fish|Ted|0.0.4
vscode-terminal-here|Tyr|0.1.1
better-comments|aar|1.2.3
project-manager|ale|0.25.2
All-Autocomplete|Ati|0.0.13
emojisense|bie|0.4.0
vscode-toggle-quotes|Bri|0.3.2
githistory|don|0.4.1
path-autocomplete|ion|1.10.0
vscode-github|Kni|0.28.1
vscode-idl|kur|0.0.3
restructuredtext|lex|65.0.0
python|ms-|2018.5.0-rc
cpptools|ms-|0.17.3
autodocstring|njp|0.2.1
psi-header|psi|1.3.6
rewrap|stk|1.9.1
code-spell-checker|str|1.6.10
selectline-statusbar|tom|0.0.2

(1 theme extensions excluded)

</details>

**Comment by albireox** [comment]
This is the entry in my keybindings file

```
    {
        "key": "shift+cmd+a",
        "command": "git.stage"
    },
```

I have tried both modifying that file directly and assigning the shortcut using the keybinding page. I have also tried a few different shortcuts.

**Comment by bpasero** [comment]
Isn't this https://github.com/Microsoft/vscode/issues/42909 ?

**Comment by albireox** [comment]
Not sure. It's a different command (stage changes vs stage selected ranges) and this fails only in Insiders. But it may be related.

**Comment by joaomoreno** [comment]
_Broken_ by 08f8a4842d752b5fb03fbf6e7e650b219f6ee9e5. I actually had code checking for that weird object `{}`, so I need some adoption here. @alexandrudima I have a feeling I could be the only extension which relied on this weird behaviour, but maybe there are others?

**Comment by bpasero** [comment]
This should maybe go into the release notes.

**Comment by joaomoreno** [comment]
@alexandrudima My fix here was to basically filter out that `null` value in the argument list: https://github.com/Microsoft/vscode/commit/30317e97bdd3efb411c1a217d1772e055e3c1edb

I will document the https://github.com/Microsoft/vscode/commit/08f8a4842d752b5fb03fbf6e7e650b219f6ee9e5 change in the release notes, since it might hit other people as well.

**Comment by albireox** [comment]
It's now working for me as well. Thanks!

**Comment by alexandrudima** [comment]
fyi @joaomoreno the correct fix landed for #42851, so `undefined` will be passed in from the keybindings case.
